<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">
QFrame {
border-radius:10px;
background-color:#1E0070;
 }
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame_superior">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border-radius:18px;
}

QPushButton:hover{
	background-color: rgb(0, 0, 0);
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>443</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="bt_minimizar">
            <property name="minimumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>image/minus.svg</normaloff>image/minus.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_maximizar">
            <property name="minimumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>image/chevron-up.svg</normaloff>image/chevron-up.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_restaurar">
            <property name="minimumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>image/chevron-down.svg</normaloff>image/chevron-down.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_cerrar">
            <property name="minimumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>image/x.svg</normaloff>image/x.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>1200</width>
               <height>1224</height>
              </rect>
             </property>
             <property name="maximumSize">
              <size>
               <width>1200</width>
               <height>16777215</height>
              </size>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QFrame" name="frame_contenedor">
                <property name="maximumSize">
                 <size>
                  <width>1300</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font-family: Noto Sans SC;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(81, 0, 135, 255), stop:0.427447 rgba(41, 61, 132, 235), stop:1 rgba(155, 79, 165, 255));
</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <property name="lineWidth">
                 <number>1</number>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_2">
                <property name="minimumSize">
                 <size>
                  <width>1600</width>
                  <height>1200</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <widget class="QPushButton" name="pushButton">
                 <property name="geometry">
                  <rect>
                   <x>110</x>
                   <y>90</y>
                   <width>151</width>
                   <height>121</height>
                  </rect>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Inventario</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="pushButton_2">
                 <property name="geometry">
                  <rect>
                   <x>360</x>
                   <y>90</y>
                   <width>151</width>
                   <height>121</height>
                  </rect>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Empleado</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="pushButton_3">
                 <property name="geometry">
                  <rect>
                   <x>610</x>
                   <y>90</y>
                   <width>151</width>
                   <height>121</height>
                  </rect>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Factura</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="pushButton_4">
                 <property name="geometry">
                  <rect>
                   <x>860</x>
                   <y>90</y>
                   <width>151</width>
                   <height>121</height>
                  </rect>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Clientes</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="pushButton_5">
                 <property name="geometry">
                  <rect>
                   <x>110</x>
                   <y>510</y>
                   <width>151</width>
                   <height>121</height>
                  </rect>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Transacciones</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="pushButton_6">
                 <property name="geometry">
                  <rect>
                   <x>860</x>
                   <y>510</y>
                   <width>151</width>
                   <height>121</height>
                  </rect>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Envíos</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="pushButton_7">
                 <property name="geometry">
                  <rect>
                   <x>610</x>
                   <y>510</y>
                   <width>151</width>
                   <height>121</height>
                  </rect>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Proyectos</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="pushButton_8">
                 <property name="geometry">
                  <rect>
                   <x>360</x>
                   <y>510</y>
                   <width>151</width>
                   <height>121</height>
                  </rect>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>G. Transacciones</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="pushButton_9">
                 <property name="geometry">
                  <rect>
                   <x>110</x>
                   <y>730</y>
                   <width>151</width>
                   <height>121</height>
                  </rect>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Inversiones</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="pushButton_10">
                 <property name="geometry">
                  <rect>
                   <x>860</x>
                   <y>730</y>
                   <width>151</width>
                   <height>121</height>
                  </rect>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Metas</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="pushButton_11">
                 <property name="geometry">
                  <rect>
                   <x>610</x>
                   <y>730</y>
                   <width>151</width>
                   <height>121</height>
                  </rect>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Tareas</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="pushButton_12">
                 <property name="geometry">
                  <rect>
                   <x>360</x>
                   <y>730</y>
                   <width>151</width>
                   <height>121</height>
                  </rect>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>G. Inversiones</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="pushButton_13">
                 <property name="geometry">
                  <rect>
                   <x>110</x>
                   <y>300</y>
                   <width>151</width>
                   <height>121</height>
                  </rect>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>G. Ingresos</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="pushButton_14">
                 <property name="geometry">
                  <rect>
                   <x>860</x>
                   <y>300</y>
                   <width>151</width>
                   <height>121</height>
                  </rect>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Proveedor</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="pushButton_15">
                 <property name="geometry">
                  <rect>
                   <x>610</x>
                   <y>300</y>
                   <width>151</width>
                   <height>121</height>
                  </rect>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Boleta PDF</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="pushButton_16">
                 <property name="geometry">
                  <rect>
                   <x>360</x>
                   <y>300</y>
                   <width>151</width>
                   <height>121</height>
                  </rect>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>G. Gastos</string>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
